rfc6962-bis.xml

```diff
@@ -366,35 +366,35 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
               Wildcard <spanx style="verb">*</spanx> labels MUST NOT be redacted, but one or more non-wildcard labels in each <xref target="RFC6125">DNS-ID</xref> can each be replaced with a redacted label as follows:
             </preamble>
             <artwork>
-REDACT(label) = prefix || BASE32(index || _label_hash)
-_label_hash = LABELHASH(keyid_len || keyid || label_len || label)</artwork>
+  REDACT(label) = prefix || BASE32(index || _label_hash)
+    _label_hash = LABELHASH(keyid_length || keyid || label_length || label)</artwork>
```

Comment:
xml2rfc complains that this line of artwork "is 75 characters; longer than 72".

I suggest: s/_length/_len/